src/spikeinterface/preprocessing/motion.py

```diff
@@ -293,86 +319,48 @@ def correct_motion(
-
-
-    Possible presets : {}
-
-    Parameters
-    ----------
-    recording : RecordingExtractor
-        The recording extractor to be transformed
-    preset : str, default: "nonrigid_accurate"
-        The preset name
-    folder : Path str or None, default: None
-        If not None then intermediate motion info are saved into a folder
-    output_motion : bool, default: False
-        It True, the function returns a `motion` object.
-    output_motion_info : bool, default: False
-        If True, then the function returns a `motion_info` dictionary that contains variables
-        to check intermediate steps (motion_histogram, non_rigid_windows, pairwise_displacement)
-        This dictionary is the same when reloaded from the folder
-    overwrite : bool, default: False
-        If True and folder is given, overwrite the folder if it already exists
-    detect_kwargs : dict
-        Optional parameters to overwrite the ones in the preset for "detect" step.
-    select_kwargs : dict
-        If not None, optional parameters to overwrite the ones in the preset for "select" step.
-        If None, the "select" step is skipped.
-    localize_peaks_kwargs : dict
-        Optional parameters to overwrite the ones in the preset for "localize" step.
-    estimate_motion_kwargs : dict
-        Optional parameters to overwrite the ones in the preset for "estimate_motion" step.
-    interpolate_motion_kwargs : dict
-        Optional parameters to overwrite the ones in the preset for "detect" step.
 
     {}
+    {}
+    raise_error : bool, default: True
+        If True, an error is raised if motion estimation fails
+        If False, the process continues and the peaks and peak_locations are still returned in `motion_info`.
 
     Returns
-    -------
-    recording_corrected : Recording
-        The motion corrected recording
-    motion : Motion
-        Optional output if `output_motion=True`.
+    =======
     motion_info : dict
-        Optional output if `output_motion_info=True`. This dict contains several variable for
-        for plotting. See `plot_motion_info()`
+        A dictionary containing a motion objects, peaks, peak locations, run_times and the parameters used to compute these.
     """
+
     # local import are important because "sortingcomponents" is not important by default
     from spikeinterface.sortingcomponents.peak_detection import detect_peaks, detect_peak_methods
     from spikeinterface.sortingcomponents.peak_selection import select_peaks
     from spikeinterface.sortingcomponents.peak_localization import localize_peaks, localize_peak_methods
-    from spikeinterface.sortingcomponents.motion import estimate_motion, InterpolateMotionRecording
     from spikeinterface.core.node_pipeline import ExtractDenseWaveforms, run_node_pipeline
+    from spikeinterface.sortingcomponents.motion import estimate_motion
+    from spikeinterface.sortingcomponents.motion.motion_estimation import estimate_motion_methods
```

Comment:
```suggestion
    from spikeinterface.sortingcomponents.motion.motion_estimation import estimate_motion, estimate_motion_methods
```
Since you are already importing from the submodule, I think this is cleaner